app/services/charges/charge_models/base_service.rb

```diff
@@ -20,6 +20,8 @@ def apply
         result.full_units_number = aggregation_result.full_units_number
         result.count = aggregation_result.count
         result.amount = compute_amount
+        result.total_aggregated_units = aggregation_result.total_aggregated_units if aggregation_result.total_aggregated_units
```

Comment:
Style/IfUnlessModifier: Modifier form of `if` makes the line too long. (https://rubystyle.guide#if-as-a-modifier)